### PR TITLE
Zenodo: Add Justin & Kale as Co-Authors

### PR DESCRIPTION
**Pull request by ax3l**
Adding @JustinRayAngus and @kale-j as a co-authors of WarpX. Thank you for your contributions! :sparkles:

Please help me to double-check your ORCiDs and spellings (e.g., inclusion of middle names, affiliations).

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -20,6 +20,11 @@
         "name": "Andriyash, Igor",
         "orcid": "0000-0003-0313-4496"
       },
+      {
+        "affiliation": "Lawrence Livermore National Laboratory",
+        "name": "Angus, Justin R.",
```
Comment (ax3l) [inline .zenodo.json]: Like this? @JustinRayAngus 
```suggestion
        "name": "Angus, Justin Ray",
```

File: .zenodo.json [hunk 2]
```diff
@@ -155,19 +160,29 @@
...
         "orcid": "0000-0001-7680-8733"
       },
       {
         "affiliation": "Modern Electron",
         "name": "Scherpelz, Peter",
         "orcid": "0000-0001-8185-3387"
       },
+      {
+        "affiliation": "University of Rochester, Laboratory for Laser Energetics",
+        "name": "Weichman, Kale",
```
Comment (ax3l) [inline .zenodo.json]: - No comma in affiliation?
- Do you prefer to include a middle name in scientific publications, @kale-j?
```suggestion
        "affiliation": "Laboratory for Laser Energetics, University of Rochester",
        "name": "Weichman, Kale",
```

Comment (kale-j) [reply .zenodo.json]: The affiliation should be "Laboratory for Laser Energetics, University of Rochester" to match the style you have for other contributors
I don't use my middle name in scientific publications, so please take out the initial

Comment (ax3l) [reply .zenodo.json]: Thank you! :pray:

File: .zenodo.json (hunk 1 above)
Comment (JustinRayAngus) [reply .zenodo.json]: Either way is fine with me. Orcid looks correct.

**Review by RemiLehe — APPROVED** [review]

**Pull request merged**